security/calico-network-policy.md

```diff
@@ -97,6 +97,12 @@ The Kubernetes network policy specification defines the following behavior:
 
 For compatibility with Kubernetes, **Calico network policy** follows the same behavior for Kubernetes pods.  For other endpoint types (VMs, host interfaces), Calico network policy is default deny. That is, only traffic specifically allowed by network policy is allowed, even if no network policies apply to the endpoint.
 
+### Before you begin
+
+Calicoctl must be **installed** and **configured** before use. Calicoctl will use etcd as the datastore by default, but many Calico installation manifests configure Kubernetes as the datastore. You can find more information on how to install and configure calicoctl in the following links:
```

Comment:
Operator install is going to replace legacy installs with manifests so recommend deleting reference to any specific install method...always use backticks for code and references to clis, and use {{site.prodname}} for any product occurrence in markdown (except title and description). Suggest this rewrite:

- Install and configure `calicoctl`
  `calicoctl` uses etcd as the default datastore, but Kubernetes is widely used as a datastore. For help, see:

- <links>
- <link>